src/Components/App.tsx

```diff
@@ -1,13 +1,143 @@
 import {Header} from './Header/Header';
 import {Body} from './Body/Body';
 import {Footer} from './Footer/Footer';
+import {useEffect, useState} from 'react';
+import {SortOptionType} from './Body/SortPanel/SortPanel';
 import './App.scss';
 
+export interface Movie {
+  id: number,
+  image: string,
+  title: string,
+  genre: string,
+  date: Date,
+  url: string,
+  rating: number,
+  runtime: number,
+  description: string
+}
+
+export interface MoviesListProps{
+  movies: Movie[],
+  onDelete: (number) => void,
+  sortMoviesHandler: (option: SortOptionType) => void
+}
+
+const moviesData : Movie[] =[
```

Comment:
I would create movie service for working with data. It could be more comfortable in the feature.